packages/cli/src/Command/Console.php

```diff
@@ -0,0 +1,225 @@
+        'html_theme_options',
+        'general',
+        'intersphinx_mapping'
+    ];
+
+    public function __construct(
+    ) {
+        parent::__construct('console');
+
+        $this->addArgument(
+            'input',
+            InputArgument::REQUIRED,
+            'Path to directory where settings.cfg is stored, and guides.xml will be generated.',
+        );
+
+        $this->addOption(
+            'force',
+            null,
+            InputOption::VALUE_NONE,
+            'When set, forces to overwrite possibly existing XML.'
+        );
+
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $settingsFile = $input->getArgument('input') . '/Settings.cfg';
+        $guidesFile = $input->getArgument('input') . '/guides.xml';
+
+        if (!file_exists($settingsFile)) {
+            $output->writeln('<error>Could not open ' . $settingsFile . '</error>');
+            return Command::FAILURE;
+        }
+
+        if (file_exists($guidesFile) && !$input->getOption('force')) {
+            $output->writeln('<error>Target file already exists in specified directory (' . $guidesFile . ')</error>');
+            return Command::FAILURE;
+        }
+
+        $output->writeln('Converting ' . $settingsFile . ' to ' . $guidesFile . ' ...');
+
+        if (!$this->convertSettingsToGuide($output, $settingsFile, $guidesFile)) {
+            $output->writeln('<error>Settings could not be converted. Please check for proper syntax.</error>');
+            return Command::FAILURE;
+        }
+
+        $output->writeln('Settings converted. You can now delete Settings.cfg and add guides.xml to your repository.');
+
+        return Command::SUCCESS;
+    }
+
+    protected function convertSettingsToGuide(OutputInterface $output, string $inputFile, string $outputFile): bool
+    {
+        // Settings.cfg can be parsed as an INI file. If it fails, bail out.
+        try {
+            $settings = @parse_ini_file($inputFile, true, INI_SCANNER_RAW);
+        } catch (\Exception $e) {
+            return false;
+        }
+
+        if (!is_array($settings)) {
+            return false;
+        }
+
+        // This array will hold all setting values that we could not migrate to guides.xml
+        $unmigratedSettings = $settings;
+
+        // Holds number of converted settings
+        $convertedSettings = 0;
+
+        // Setup basic DOM Document
+        $dom = new \DOMDocument("1.0", "UTF-8");
+        $dom->preserveWhiteSpace = false;
+        $dom->formatOutput = true;
+
+        // Add static <guides> element with proper XMLNS
+        $guides = $dom->createElement('guides');
+        $guides->setAttribute('xmlns', 'https://www.phpdoc.org/guides');
+        $guides->setAttribute('xmlns:xsi', 'http://www.w3.org/2001/XMLSchema-instance');
+        $guides->setAttribute('xsi:schemaLocation', 'https://www.phpdoc.org/guides vendor/phpdocumentor/guides-cli/resources/schema/guides.xsd');
```

Comment:
```suggestion
        $guides->setAttribute('xsi:schemaLocation', 'https://www.phpdoc.org/guides ../vendor/phpdocumentor/guides-cli/resources/schema/guides.xsd');
```

As it is available in Documentation folder, so one level up. But this may also not work, many extensions define a .Build/vendor folder. But as we don't know this, that is fine for now. Maybe we should hint a message about it at the end.